universal-application-tool-0.0.1/test/controllers/HomeControllerAuthenticatedTest.java

```diff
@@ -40,10 +65,7 @@ public void testAuthenticatedSecurePage() {
         fakeRequest(routes.HomeController.secureIndex())
             .header(Http.HeaderNames.HOST, "localhost:" + testServerPort());
     Result result = route(app, request);
-    // This should work, but it doesn't!
-    // assertThat(result.status()).isEqualTo(HttpConstants.OK);
-
-    // This is the opposite of what we want, but it's what happens now anyway.
-    assertThat(result.status()).isEqualTo(HttpConstants.UNAUTHORIZED);
+    Logger.of("application").info(result.headers().toString());
```

Comment:
suggestion (blocking): instead of logging here, can we assert something about the header?

Reply:
I actually left that in there by accident, removed!